proto/toit/api/user.proto

```diff
@@ -106,6 +106,8 @@ message Invoice {
     string pdf_url = 4;
     string status = 5;
     google.protobuf.Timestamp due_date = 6;
+    string receipt_pdf_url = 7;
+    int64 amount_paid = 8;
```

Comment:
should this be a float or double?

Reply:
The amount is in cents, i.e. the amount paid is equal to 4000 for a 40$ invoice